homeworks/dankhazipeter/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
feltettem egy plugint, ami mindent aláhúz, ha nem tartom be a standardeket, ezért tettem dokumentációt a kód elejére a függvényhez is :)

Ezt szerintem a jövőben majd nem csinálom ennyire rendesen, mert méteres lesz a PR, ami Neked nem biztos, hogy jó. Vagy írjam oda nyugodtan ezeket?